frontends/web/src/routes/buy/pocket.tsx

```diff
@@ -96,6 +97,18 @@ export const Pocket = ({ code }: TProps) => {
             }
           });
       }
+      if (message.type === V0MessageType.VerifyAddress) {
+        setVerifying(true);
+        verifyAddress('p2wpkh', message.bitcoinAddress, code)
+          .then(response => {
+            setVerifying(false);
+            if (response.status === 'used-address') {
+              // This should not happen, unless the user receives a tx on the same address between the message signing
+              // and the address verification.
+              alertUser(t('buy.pocket.usedAddress.err1') + message.bitcoinAddress + t('buy.pocket.usedAddress.err2'));
```

Comment:
instead of 2 different locize keys, there is a way to add parameter to 1 key, see:
- https://github.com/digitalbitbox/bitbox-wallet-app/blob/82b3291732d0e5030de43f1217caeadb2d84f91b/frontends/web/src/locales/en/app.json#L34
- https://github.com/digitalbitbox/bitbox-wallet-app/blob/82b3291732d0e5030de43f1217caeadb2d84f91b/frontends/web/src/routes/account/info/info.tsx#L77-L81